data/tlg0062/tlg054/tlg0062.tlg054.perseus-eng3.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+    <teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="eng">The Dipsads</title>
+<title xml:lang="grc">Περὶ τῶν διψάδων</title>
```

Comment:
We don't tend to include both English and Greek titles in translation file, just typically the translation title.